libvcx/src/api/vcx.rs

```diff
@@ -15,17 +15,117 @@ use crate::utils::error;
 use crate::utils::threadpool::spawn;
 use crate::utils::version_constants;
 
-/// Initializes VCX with config settings
+#[no_mangle]
+pub extern fn vcx_init_threadpool() -> u32 {
```

Comment:
Let's add json config parameter and let enable use to specify number of threads

Reply:
Perhaps later we can add other options to the config file, if we'd want to give user more options over threadpool configuration